src/Features/CSharp/Portable/SignatureHelp/TupleConstructionSignatureHelpProvider.cs

```diff
@@ -24,7 +24,7 @@ internal class TupleConstructionSignatureHelpProvider : AbstractCSharpSignatureH
         private static readonly Func<TupleExpressionSyntax, SyntaxToken> s_getOpenToken = e => e.OpenParenToken;
         private static readonly Func<TupleExpressionSyntax, SyntaxToken> s_getCloseToken = e => e.CloseParenToken;
         private static readonly Func<TupleExpressionSyntax, IEnumerable<SyntaxNodeOrToken>> s_getArgumentsWithSeparators = e => e.Arguments.GetWithSeparators();
-        private static readonly Func<TupleExpressionSyntax, IEnumerable<string>> s_getArgumentNames = e => e.Arguments.Select(a => a.NameColon?.Name.Identifier.ValueText ?? string.Empty);
+        private static readonly Func<TupleExpressionSyntax, IEnumerable<string>> s_getArgumentNames = e => e.Arguments.Select(a => a.NameColon?.Name.Identifier.ValueText);
```

Comment:
I don't understand why we can delete the `?? string.Empty` here, since we still do a conditional access for `a.NameColon?.Name`

Reply:
great catch!